Settings: Document the new Settings UI/UX (#240)
exoplatform/exo-documentation: docs/GettingStarted.rst

 * :ref:`Digital Workplace Widgets <DW-Widgets>`
   Introduction to widget blocks that come with the Digital Workplace.

 * :ref:`Settings <Settings>`

>    Instructions on how to configure the UI language, Timezone, notifications and password of eXo Platform.

Reviewer: ```suggestion
   Instructions on how to configure your UI language, Timezone, notifications and password.
```

.. _Change-Language:

Settings
~~~~~~~~~~

The platform settings allow you to configure various settings for your Digital Workplace experience like: 

> - Choose your language 

Reviewer: ```suggestion
- Choose the language in which you want to view the user interface
```

        It means the language set by the user will be at the highest level, and the site's language at the lowest level.
        Accordingly, you should pay attention to this order when selecting your preferred display language.

TimeZone
-----------

> To change the Timezone of eXo Platform used for the Calendar, do as follows:

Reviewer: ```suggestion
To change the Timezone of to be used for events in the eXo Calendar, do as follows:
```

1. Click on edit Timezone icon.

2. A drawer that contains Timezones that eXo Platform supports. 

3. Select your preferred Timezone to use:

|settings3|

3. Click Apply to commit your changes.

Notifications
---------------

> eXo Platform provides you a complete notification function which helps you to

Reviewer: ```suggestion
eXo Platform provides an extensive notification system which helps you to
```

> avoid missing anything in your organization. As from eXo Platform 5.1, there

Reviewer: ```suggestion
avoid missing anything in your organization. There
```

> are 3 notification types that you can use.

Reviewer: ```suggestion
is three types of notifications by default
```

|settings4|

You can select notification options as desired. In particular:

> 

Reviewer: something missing here?

Allows to enable or disable one or namy notifications channels by switching the button between Yes and No:

-  **Notify me by email**: If you choose No option, the functions 
   concerning email notification will be hidden.

-  **Notify me on-site**: If you choose No option, the functions 
   concerning on-site notification will be hidden.

-  **Notify me on mobile**: If you choose No option, the functions
   concerning mobile notifications will be hidden.  

> .. note:: - A switch button is available to enable or disable the notification type.

Reviewer: ```suggestion
.. note:: - A switch button lets you enable or disable the notification type.
```

>           - A descriptiion is available below every notification type to give more information about the notification type.

Reviewer: ```suggestion
          - A description is available below every notification.
```

.. _On-site:

**On-site**

> This real-time notification helps you receive new information without a

Reviewer: ```suggestion
real-time notifications are displayed in the notification center inside eXo Platform. A bell icon with a ticker indicating the number of unread notifications is updated dynamically eaxh time you receive a notification without
```

> browser refresh. Whenever there is a new activity happening within your

Reviewer: ```suggestion
refreshing your browser.
```

> network, it will be pushed into your notifications menu accompanied by

Reviewer: ```suggestion
```

> the number of unread messages as follows:

Reviewer: ```suggestion
```

|settings5|

> When you click the Notification icon, all notifications will be listed in a drawer 

Reviewer: ```suggestion
When you click the Notification icon, all notifications are listed in a drawer 
```

starting from the most recent one:

|settings6|

By clicking each notification item, you will be redirected to the

> corresponding activity stream or you can select **View All** to see all

Reviewer: ```suggestion
corresponding activity or any other object that triggered the notification (such a a task, a news, etc.). Click **View All** to see all
```

> notifications. Besides, the **Mark all as read** function allows you to

Reviewer: ```suggestion
notifications. Besides, the **Mark all as read** function lets you
```

change all messages on this menu into the read status. Finally, to

> remove any notification item, simply click the corresponding |image51|.

Reviewer: ```suggestion
remove any notification item, simply click the corresponding close arrow |image51|.
```

> .. note:: You can also view all your notifications in the page ``My Notifications`` accessible by clicking on

Reviewer: duplicate content

          your See all on the bottom notifications drawer :

            |settings7|

>             

Reviewer: duPlicate content

          It is possible to mark all your unread notifications as read simply by cliking on ``Mark all as read`` on the bottom notifications drawer:
          
            |settings8|
            

> .. tip:: When you are not on the platform web page and you receive On-site notifications, 

Reviewer: duplicate content

         to grab your attention, a number appears in the web browser tab indicating the number of unread
         notifications.
         
         If the platform browser tab is pinned, an indication appears near the favicon to attract 
         your attention about unread notifications.
               
                 |image96|
 

.. _mobilePushNotifs:

> **Mobile push notifications**

Reviewer: We should indicate that push notifications channel will be available if the administrator has enabled and setup push notifs and then link to the admin screen where this can be configured

A push notification is a real-time notification that pops up on a mobile
device (iOS and Android). It is pushed instantly when the action is 
done.

The push notification displays the same content as the on-site 
notification. It also displays the eXo Mobile application logo, the 
site's name (eXo community for example) and the receipt time. 
Clicking on it opens directly the concerned activity.

|pushNotif1|

.. _Via-emails:

**Via emails**

> Besides on-site notifications, you are totally able to keep track of

Reviewer: ```suggestion
Besides on-site and push notifications, you can keep track of
```

activities and events via emails. In eXo Platform, it is easy to control your
own email notifications from a single location in the user settings.
When this function is enabled, you will receive emails in 2 ways:

-  A notification email with different content for each event type. For
   example, for the activity embedding a video, the message will be
   represented as a thumbnail image of the embedded media, like below:

   |image52|

   By clicking the **Watch the video** link, you will be redirected to
   the activity stream where you can play the embedded video.

-  A digest email that collects all notifications during a certain
   period and is sent once per day or per week.

   |image53|

    .. note:: To receive notification and digest emails as well as on-site notifications, your administrator must enable notification plugins first. See :ref:`Notification administration <NotificationAdministration>` for details.

This section shows you how to manage your notifications via the
**Manage Notifications** view.

Manage Notifications 
~~~~~~~~~~~~~~~~~~~~~~~~

> By clicking on Manage Notifications, a new view is displayed that contains all notification you have in the 

Reviewer: ```suggestion
By clicking on Manage Notifications, a new view is displayed that contains all kinds of notifications that are possible to receive in the 
```
